scripts/generate/index.ts

```diff
@@ -34,6 +35,8 @@ async function init({type}: InitConfig) {
     })
   );
 
+  await tryCatch(() => generateEnv({rootDir}));
```

Comment:
Do you think it's necessary to do this as part of the generator? I am not sure this will be common enough that it needs to be part of the default project template, I think it's something we can just document and use if it's there.